frontend/src/store/index.js

```diff
@@ -25,6 +25,144 @@ Vue.use(Vuex)
 export const store = new Vuex.Store({
 
   state: {
+    addIndividualITAsteps: [
+      {
+        step: 1,
+        title:'Exam Type',
+        questions: [
+          {
+            key: 'exam_type_id',
+            text: 'Exam Type ID / Colour',
+            kind:'dropdown',
+            minLength: 0,
+            digit: false,
+          }
+        ]
+      },
+      {
+        step: 2,
+        title:'Exam Info',
+        questions: [
+          {
+            key: 'event_id',
+            text:'Event ID' ,
+            kind: 'input',
+            minLength: 6,
```

Comment:
Why the `minLength` attributes? I can imagine this is going to be a painpoint

Reply:
it can be set to zero and then it doesn't get used.  i figured at least with the ITA exams this would be a good thing - I'm sure the eventID they supply is always the same number of digits so if it is entered with any less it would be guaranteed to be invalid.